gravity_upgrade.md

```diff
@@ -0,0 +1,101 @@
+# Gravity Upgrade Training (for Gravity 5.5 and earlier)
+
+## Introduction
+
+_Note: This part of the training pertains to Gravity 5.5 and earlier._
+
+This part of the workshop focuses on Gravity upgrades, what is happening under the hood, how to debug and troubleshoot failures, and many more situations. Cluster upgrades can get complicated for complex cloud apps composed of multiple microservices, the goal of this training will be to first build a foundational understanding of how Gravity handles upgrades, step through a manual upgrade example, and then go through some real-world upgrade scenarios.
+
+## Gravity Upgrade Overview
+
+Cluster upgrades require two major layers that need updating. The first layer is k8s itself and its dependencies, and the second layer is the applications deployed inside.
+
+The Gravity upgrade process handles both of these mentioned layers in the following manner:
+
+1. A new version of an application and container images are downloaded and stored inside the cluster and transferred onto a cluster node. 
+
+2. Gravity uses k8s rolling update mechanism to perform the update.
+
+3. If applicable, before or after the upgrade custom update hooks can be used to perform application specific actions.
+
+Upgrades can be triggered both via the Gravity Hub (Ops Center) or from the CLI. Below we will go over the upgrade via the CLI to give some detail to the above steps mentioned.
+
+### Step 1: Providing new version of application
+
+As mentioned the first step in the upgrade process is to import your new application bundle onto a Gravity cluster and transfer it onto a cluster nodes to execute the upgrade command.
+
+Upgrades can be handled both in online and offline environments.
+
+#### Online Gravity Upgrade
+
+In the case a Gravity cluster is connected to the Gravity Hub (Ops Center) and it has remote support turned on, you can download an updated application bundle directly from the Ops Center. 
+
+Using the `gravity update download` you can check and download any updates
+
+#### Offline Gravity Upgrade
+
+In the case where a Gravity cluster is offline the new version of the application bundle will need to be copied over to one of the application cluster nodes that is accessible to the other nodes.
```

Comment:
"application bundle" -> "cluster image", and same comment as above about the tarball placed on a master node.